### DISCUSSION TITLE
Alpine Vaultwarden wants password in Shell although auto-login is set

### DISCUSSION THREAD

**Discussion by johndoe0815** [tteck/Proxmox#1268] [Help Needed] [answered]
I have installed alpine vaultwarden with default settings and during install it also says

_Using Default Settings
Using Container Type: 1
Using Root Password: Automatic Login
Using Container ID: 101
Using Hostname: alpine-vaultwarden
Using Disk Size: 0.3GB_

Nevertheless, when I open the shell of the container, it prompts for a password:

`Welcome to Alpine Linux 3.17
Kernel 5.15.83-1-pve on an x86_64 (/dev/tty1)

alpine-vaultwarden login: `

What is that password and why is the auto-login not working, like I have it for all other containers from your site?

**Comment by tteck** [comment, marked as answer]
The Alpine autologin feature simply means that there is no root password required for logging in, but you still need to enter the username "root" manually.
